src/man/sssd-simple.5.xml

```diff
@@ -40,25 +40,27 @@
             apply:
             <itemizedlist>
                 <listitem>
-                    <para>If all lists are empty, access is granted</para>
+                    <para>If all lists are empty, access is granted.</para>
                 </listitem>
                 <listitem>
                     <para>
-                        If any list is provided, the order of evaluation is
-                        allow,deny. This means that any matching deny rule
+                        If any list is provided, the order of evaluation is:
+                        allow → deny. This means that any matching deny rule
                         will supersede any matched allow rule.
                     </para>
                 </listitem>
                 <listitem>
                     <para>
                         If either or both "allow" lists are provided, all
-                        users are denied unless they appear in the list.
+                        users are denied unless they appear in **at least one**
```

Comment:
I don't think the surrounding ** is needed here, but I don't insist.

Reply:
I don't have a strong opinion either, just removed.